block/src/Pos/Block/Pure.hs

```diff
@@ -58,14 +58,22 @@ data VerifyHeaderParams = VerifyHeaderParams
       -- ^ Check that header has no unknown attributes.
     } deriving (Eq, Show)
 
+maybeMempty :: Monoid m => (a -> m) -> Maybe a -> m
```

Comment:
One can't just put random undocumented unused utility function in the middle of the module. This is related to `verifyFromEither` as well, it's only used in `verifyHeader` and it's small so it should be in `where` case. It also completely discards `Left`, while it can be put into `VerFailure`.

Reply:
This function was added in 2016 and IIRC it was removed recently by @thatguy in one of his famous _Cosmetic changes_ PRs.